src/tracer.ts

```diff
@@ -0,0 +1,12 @@
+import { getTracer, startTracer, Tracer } from "@fwl/tracing";
+
+startTracer({
+  simpleCollector: {
+    serviceName: "connect-account",
+    url: "http://localhost:9411/api/v2/spans",
```

Comment:
both of these should come from the config

Reply:
sorry, the whole `simpleCollector` should come from the config :/

Reply:
b74e847

Looks like we can only use public Satellites with Lightstep, so here is my take. I will document it if this is good enough.